### ISSUE TITLE
Is there a example on how to use quantization-aware training ?

### ISSUE THREAD

**Issue by Zehaos** [NervanaSystems/distiller#154]
Is there a example on how to use quantization-aware training ?

**Comment by guyjacob** [comment]
Yes, there are a few under [examples/quantization/quant_aware_train].(https://github.com/NervanaSystems/distiller/tree/master/examples/quantization/quant_aware_train)

**Comment by guyjacob** [comment]
Closing this, feel free to re-open if there are any more questions

**Comment by SystemErrorWang** [comment]
@guyjacob would like to know, if we can ues "distiller.file_config" and "distiller.file_config" to load the .yaml config file and run quantization aware training?

**Comment by guyjacob** [comment]
I'm not sure what you refer to by "distiller.file_config". Can you clarify?

**Comment by SystemErrorWang** [comment]
@guyjacob 
i am sorry to have made a typo, it should be "distiller.CompressionScheduler" and "distiller.file_config"
for example, i saved my training settings in "train_config.yaml", then i run my training code like this:

`compression_scheduler = distiller.CompressionScheduler(model) 
    compression_scheduler = distiller.file_config(model, optimizer, 
                                    '/home/username/distiller/workingdir/train_config.yaml', 
                                    compression_scheduler)
    for epoch in range(total_epoch):
        compression_scheduler.on_epoch_begin(epoch)
        for train_step, (inputs, target) in enumerate(train_loader):
            total_iter += 1
            inputs, target = inputs.cuda().float(), target.cuda().float()
            compression_scheduler.on_minibatch_begin(epoch, train_step, 
                                                    num_iter, optimizer)
            output = model(inputs)
            loss = criterion(output, target)
            agg_loss = compression_scheduler.before_backward_pass(epoch, train_step, 
                                                        num_iter, loss, optimizer, True)
            loss = agg_loss.overall_loss
            optimizer.zero_grad()
            loss.backward()
            optimizer.step()
            compression_scheduler.on_minibatch_end(epoch, train_step, 
                                                num_iter, optimizer)`

**Comment by guyjacob** [comment]
Oh OK, I understand. So yes - that is the way to create a scheduler for quantization aware training.

**Comment by SystemErrorWang** [comment]
@guyjacob  thank you so much for your kind help!
